docs/conf.py

```diff
@@ -24,13 +24,17 @@
     "sphinx.ext.viewcode",
     'python_docs_theme',
     'sphinx_markdown_tables',
+    'sphinx_copybutton',
 ]
 
 templates_path = ['_templates']
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
 language = 'go'
 
+# https://sphinx-copybutton.readthedocs.io/en/latest/use.html#strip-and-configure-input-prompts-for-code-cells
+copybutton_prompt_text = ">>> "
```

Comment:
Why not use https://sphinx-copybutton.readthedocs.io/en/latest/use.html#automatic-exclusion-of-prompts-from-the-copies :
```
copybutton_exclude = '.linenos, .gp, .go'
```
Then you can put `$` in bash prompts instead of `>>>`, which is a python console.